mmv1/third_party/terraform/services/compute/resource_compute_instance.go.tmpl

```diff
@@ -548,6 +556,16 @@ func ResourceComputeInstance() *schema.Resource {
 							DiffSuppressFunc: tpgresource.CompareSelfLinkOrResourceName,
 							Description:      `The name or self_link of the disk attached to this instance.`,
 						},
+
+						"force_attach": {
+							Type:             schema.TypeBool,
+							Optional:         true,
+							Default:          false,
+							AtLeastOneOf:     bootDiskKeys,
+							ForceNew:         true,
```

Comment:
If forceNew is set to true, a change on this field will cause a new resource creation, is that something expected?

Reply:
It's expected. forceAttach is an `input only` field

Reply:
By `input only`, do you mean this field is `immutable` right?

Reply:
Yes this field is `immutable`.